draft-irtf-panrg-path-properties.md

```diff
@@ -196,9 +200,13 @@ Service function:
 : A service function that a path element applies to a flow, see {{RFC7665}}. Examples of abstract service functions include firewalls, Network Address Translation (NAT), and TCP optimizers. Some stateful service functions, such as NAT, need to observe the same flow in both directions, e.g., by being an element of both the path and the reverse path.
 
 Transparency:
-: A node is transparent with respect to a protocol header, payload, or both for a specific action if the node performs this action independently of the given (meta-)information.
+: A node is transparent to a flow with respect to some (meta-)information for an action if the node performs this action independently of the given (meta-)information.
+(Meta-)information can for example be the existence of a protocol (header) in a packet or the content of a protocol header, payload, or both.
 Actions can for example be blocking packets or reading and modifying (other protocol) headers or payloads.
-An IP router could be transparent for transport protocol headers such as TCP and UDP, in contrast to a NAT that actively modifies TCP and UDP header information.
+An IP router could be transparent to transport protocol headers such as TCP/UDP but not transparent to IP headers as it actively modifies them.
+A TCP SYN flooding firewall that drops TCP SYN packets after reaching a limit is transparent to IP but not to TCP/UDP.
+And a NAT that actively modifies IP and TCP/UDP header information based on their content is not transparent to either IP or TCP/UDP headers.
+Note that according to this definition, a node that modifies packets in accordance with the hosts, such as a transparent HTTP proxy, as defined in {{RFC2616}}, and a node listening and acting to implicit or explicit signals, see {{RFC8558}}, would not be considered transparent.
```

Comment:
Nit, I don't think we need the hypothetical here, and the node is reacting, right? Or maybe "acting on"?
```suggestion
Note that according to this definition, a node that modifies packets in accordance with the hosts, such as a transparent HTTP proxy, as defined in {{RFC2616}}, and a node listening and reacting to implicit or explicit signals, see {{RFC8558}}, are not considered transparent.
```